.github/workflows/dotnet-ci.yml

```
@@ -71,3 +71,23 @@
         run: dotnet sonarscanner end /d:sonar.token="${{ secrets.SONAR_TOKEN }}"
+
+  deploy:
+    name: Deploy to Railway
+    needs: build-and-analyze
+    runs-on: ubuntu-latest
+    if: always()
+    container: ghcr.io/railwayapp/cli:latest
+    
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)